docs/pages/getting-started/linux-server.mdx

```diff
@@ -166,12 +166,26 @@ On your Linux machine, run the following command to start the `teleport` daemon
   </TabItem>
 
   <TabItem label="Source or custom install">
+  1. Create a systemd unit file using the `teleport install systemd` command and the [unit file path](https://www.freedesktop.org/software/systemd/man/systemd.unit.html) you'd
+  like to use for your configuration. The `/etc/systemd/system/teleport.service` path is recommended in most use case:
```

Comment:
```suggestion
  like to use for your configuration. We recommend `/etc/systemd/system/teleport.service` path in most use cases:
```

Reply:
style nit: fixed the typo and switched to active voice.

Reply:
Switched to the active voice here and in more places, fixed typo